### PR TITLE
Standardize naming, create content for resources/index.md, and make m…

### PR DESCRIPTION
**Pull request by JillHeaden**
…inor grammatical changes.

### CODE REVIEW

File: src/Documentation/resources/ideas_for_contributions.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ title: Ideas for Contributions
 
 # Ideas for Contributions
```
Comment (sergeybykov) [inline src/Documentation/resources/ideas_for_contributions.md]: I think we should delete this page altogether. Some of the original ideas have already been implemented, some are outdated. I'm afraid this list is rather misleading now.

**Comment by sergeybykov** [comment]
It needs to be updated after #4979 got merged.

**Comment by JillHeaden** [comment]
I have made more changes and verified that "Ideas for Contributions" does not show up in my locally deployed version. 
I may need help making sure this PR is updated, if it is not happening correctly.

**Comment by JillHeaden** [comment]
Fixes #4717

**Comment by sergeybykov** [comment]
I fixed, rebased, and merged these changed directly because I couldn't fix this PR. So it is effectively merged now.

**Pull request closed without merging**